templates/OpenEdx/0/docker-compose.yml

```diff
@@ -0,0 +1,125 @@
+db:
+    image: mysql:5.7
+    labels:
+        io.rancher.sidekicks: dbConfig
+    environment:
+        MYSQL_ROOT_PASSWORD: password
+        MYSQL_DATABASE: edxapp
+        MYSQL_USER: edxapp001
+        MYSQL_PASSWORD: password
+    volumes:
+     - ${EDX_MYSQL_DATA_ROOT}:/var/lib/mysql
+    volume_driver: ${volume_driver}
+    
+dbConfig:
+    image: mysql:5.7
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    command: |
+      bash -c "sleep 2m; mysql -uroot -ppassword -h db -e \"create database edxapp_csmh;\""
+
+mongo:
+    labels:
+        io.rancher.sidekicks: mongoConfig
+    image: mongo:3.3
+    volumes:
+     - ${EDX_MONGO_DATA_ROOT}:/data/db
+    volume_driver: ${volume_driver}
+
+mongoConfig:
+    image: mongo:3.3
+    labels:
+        io.rancher.container.hostname_override: container_name
+        io.rancher.container.start_once: true
+    command: |
+      bash -c "sleep 2m; echo 'db.createUser({ user: \"edxapp\", pwd: \"password\", roles: [ { role: \"readWrite\", db: \"edxapp\" } ] });' > createEdxappUser.js; mongo mongo/edxapp createEdxappUser.js; echo 'db.createUser({ user: \"cs_comments_service\", pwd: \"password\", roles: [ { role: \"readWrite\", db: \"cs_comments_service\" } ] });' > createForumUser.js; mongo mongo/cs_comments_service createForumUser.js"

```

Comment:
@borisno2 same comment as above, make it user configurable rather than hard coding?